fladgejt/webui/studium.py

```diff
@@ -267,21 +267,40 @@ def create_zapisny_list(self, studium_key, akademicky_rok, rok_studia):
         app = self._open_administracia_studia()
 
         self.__open_novy_zapisny_list_dialog(app, studium_key)
+        
+        message = None
 
         # V combo boxe vyberieme rok studia.
-        app.d.rocnikComboBox.select(find_option(app.d.rocnikComboBox.options, id=str(rok_studia)))
+        if rok_studia is not None:
+            app.d.rocnikComboBox.select(find_option(app.d.rocnikComboBox.options, id=str(rok_studia)))
 
         # Ak je nastaveny akademicky rok, vyberieme ho v combo boxe.
         if akademicky_rok is not None:
-            app.d.rokComboBox.select(find_option(app.d.rokComboBox.options, id=akademicky_rok))
-
-        # Stlacime tlacidlo "OK".
-        with app.collect_operations() as ops:
-            app.d.enterButton.click()
-
-        # Dialog sa zavrie.
-        app.awaited_close_dialog(ops)
-
+            try:
```

Comment:
Je naschval, ze vsetok tento kod je vnutri `if akademicky_rok is not None`? Vyzera ze ak akademicky_rok je None, ani len nestlacime "OK".

Reply:
ok, opravene, bola to chyba